doc/user/olm-catalog/operator-bundles.md

```diff
@@ -0,0 +1,23 @@
+# Operator Bundles
```

Comment:
IMO users which will bee looking for it will try to search for something such as: "How to build Operator Registry Bundles with/for SDK" So, WDYT about something such as:

```suggestion
# Registry Bundles for Operator SDK Operators
```

See: https://github.com/operator-framework/operator-sdk/issues/1905